lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -337,49 +342,17 @@ export abstract class ClientApplication {
 
     // #region Popup Flow
 
-    /**
-     * Use when you want to obtain an access_token for your API via opening a popup window in the user's browser
-     * @param {@link (PopupRequest:type)}
-     *
-     * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
-     */
-    acquireTokenPopup(request: PopupRequest): Promise<AuthenticationResult> {
-        try {
-            this.preflightBrowserEnvironmentCheck(InteractionType.Popup);
-        } catch (e) {
-            // Since this function is synchronous we need to reject
-            return Promise.reject(e);
-        }
-
-        // asyncPopups flag is true. Acquires token without first opening popup. Popup will be opened later asynchronously.
-        if (this.config.system.asyncPopups) {
-            return this.acquireTokenPopupAsync(request);
-        } else {
-            // asyncPopups flag is set to false. Opens popup before acquiring token.
-            const popup = PopupHandler.openSizedPopup();
```

Comment:
Popups opened by the broker should also include this logic.

Reply:
We will need to add that property to the broker options then, since the broker will need to specify the popup in constructor.

Reply:
Will resolve in a separate PR once the fixes for popup flow has been merged to dev